README.md

```diff
@@ -1,62 +1,549 @@
-hypertable
-==============================================================================
+# Hypertable
 
 To follow the early life of this project, see
 [oss-components#pn-flexbox-tables](https://github.com/upfluence/oss-components/pull/54/files)
```

Comment:
We could drop this part and replace it w/ what @Miexil mentioned here: https://github.com/upfluence/hypertable/pull/259/files#r2454036574.